test/logs.spec.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import {
+  render,
+  findByText,
+  fireEvent,
+  waitForElementToBeRemoved,
+} from "@testing-library/react";
+import {
+  Arc,
+  DevArcConfig as arcConfig,
+  DAO,
+  DAOData,
+  Token,
+  TokenData,
+  Tokens,
+  Members,
+  Member,
+  MemberData,
+  ArcConfig,
+} from "../src";
+
+describe("Components with logs ", () => {
+  it("Protocol shows error", async () => {
+    const { findByTestId, container } = render(
+      <Arc config={{ ...arcConfig, graphqlHttpUrl: "wrong url" } as ArcConfig}>
+        <Tokens>
+          <Token.Data>
+            {(token: TokenData) => (
+              <div>{"Token address: " + token.address}</div>
+            )}
+          </Token.Data>
+        </Tokens>
+      </Arc>
+    );
+    const loader = await findByTestId("default-loader");
+    fireEvent.mouseEnter(loader);
+    const arcConnectionError = await findByText(
+      container,
+      /No connection to the graph/
+    );
+    expect(arcConnectionError).toBeInTheDocument();
+  });
+
+  it("ComponentList shows error", async () => {
```

Comment:
Should we add test for every component list logs? (Votes, reputations, etc etc). I'd say that we shouldn't, since this is a generic behaviour... Would like to confirm with you tho @dOrgJelli

Reply:
Yah I agree, I think it's generic and that we can write a test that can easily iterate through all types and make sure they all have Logs exposed. So one test that validates all classes' generic behavior.